app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list[Person]:
+    person_list = []
+
+    for person_dict in people:
+        person = Person(name=person_dict["name"], age=person_dict["age"])
+        person_list.append(person)
+
+    for person_dict in people:
+        person = Person.people[person_dict["name"]]
+        if "wife" in person_dict and person_dict["wife"]:
+            person.wife = Person.people[person_dict["wife"]]
+        elif "husband" in person_dict and person_dict["husband"]:
+            person.husband = Person.people[person_dict["husband"]]
```

Comment:
Similar to the previous comment, the `husband` attribute is not defined in the `Person` class. Ensure that attributes are properly initialized or handled to avoid attribute errors.